src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md

```diff
@@ -13,17 +13,17 @@ With NFTs bringing blockchain into the public eye, now is an excellent opportuni
 
 Alchemy is extremely proud to be powering the biggest names in the NFT space, including Makersplace (recently set a record digital artwork sale at Christie’s for $69 Million), Dapper Labs (creators of NBA Top Shot & Crypto Kitties), OpenSea (the world’s largest NFT marketplace), Zora, Super Rare, NFTfi, Foundation, Enjin, Origin Protocol, Immutable, and more.
 
-In this tutorial, we will walk through creating and deploying an ERC-721 smart contract on the Ropsten test network using [MetaMask](https://metamask.io/), [Solidity](https://docs.soliditylang.org/en/v0.8.0/), [Hardhat](https://hardhat.org/), [Pinata](https://pinata.cloud/) and [Alchemy](https://dashboard.alchemyapi.io/signup) (don’t fret if you don’t understand what any of this means yet — we will explain it!).
+In this tutorial, we will walk through creating and deploying an ERC-721 smart contract on the Ropsten test network using [MetaMask](https://metamask.io/), [Solidity](https://docs.soliditylang.org/en/v0.8.0/), [Hardhat](https://hardhat.org/), [Pinata](https://pinata.cloud/) and [Alchemy](https://alchemy.com/signup/eth) (don’t fret if you don’t understand what any of this means yet — we will explain it!).
```

Comment:
Hey @btandowsky, curious where you got this url? I can't replicate it by navigating through Alchemy's sign up process. The current URL seems to be working fine. Is the /eth/ some sort of tracking? If so, we definitely don't want this.

Reply:
@minimalsm It's a new URL structure that's being introduced, but the current one does work just fine and will be backwards compatible or maintained. Totally aligned with not introducing tracking links! We can reject this update and I'll just ensure we always continue to have functioning URLs in the documentation. Thanks for the help here!